data-raw/data_sources.py

```diff
@@ -0,0 +1,62 @@
+# SPDX-FileCopyrightText: 2025-present Imperial College of Science and Technology
+#
+# SPDX-License-Identifier: MIT
+
+"""
+Generate package data.
```

Comment:
Can you include somewhere (e.g., the README) the correct command to invoke `uv` to run this command please?